examples/basic/package.json

```
@@ -11,7 +11,10 @@
     "@repo/eslint-config": "workspace:*",
     "@repo/typescript-config": "workspace:*",
     "prettier": "^3.1.0",
-    "turbo": "latest"
+    "turbo": "^1.10.16"
   },
-  "packageManager": "pnpm@8.9.0"
+  "packageManager": "pnpm@8.9.0",
+  "engines": {
+    "node": "18"
```

Let's include a range so users with strict engines and 20 etc. don't get a warning:
```suggestion
    "node": ">=18"
```

---

TIL